taxi/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Manufacturer)
```

Comment:
looks right) but I did not immediately find in the previous committees portable modules

You upload many unnecessary file,
create .gitignore and writing there are all unnecessary folders there and files , and repeat correct push